pkg/providers/common/config/audit-policy.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: audit.k8s.io/v1
+apiVersion: audit.k8s.io/v1beta1
```

Comment:
where is this yaml file being used now?

Reply:
For all versions before 1.24. This will prevent the rollout of nodes.